components/eamxx/src/physics/share/physics_constants.hpp

```diff
@@ -80,6 +80,8 @@ struct Constants
   static constexpr Scalar bcn           = 2.;
   static constexpr Scalar dropmass      = 5.2e-7;
   static constexpr Scalar NCCNST        = 200.0e+6;
+  static constexpr Scalar kc            = 9.44e+9;
+  static constexpr Scalar kr            = 5.78e+3;
```

Comment:
should we change the name, maybe add a `sb_` prefix, to indicate that these values are only for the SB scheme?